Dockerfile

```diff
@@ -1,3 +1,38 @@
-# All Dockerfiles should start from this base image
-# Provide the TAG environment variable, or replace with the image version required
-FROM ghcr.io/opensafely-core/base-docker:$TAG
+FROM rocker/rstudio:4.0.5
+
+LABEL org.opencontainers.image.source https://github.com/opensafely/research-template
+
+# we are going to use an apt cache on the host, so disable the default debian
+# docker clean up that deletes that cache on every apt install
+RUN rm -f /etc/apt/apt.conf.d/docker-clean
+
+# Install python 3.10. This is the version used by the python-docker 
+# image, used for analyses using the OpenSAFELY pipeline.
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN --mount=type=cache,target=/var/cache/apt \
+    echo "deb http://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu focal main" > /etc/apt/sources.list.d/deadsnakes-ppa.list &&\
```

Comment:
I think these check failures are appearing because they're warning/info level rather than errors. Although if I've done something silly in the config please let me know.